src/core/generator/folderGenerator.ts

```diff
@@ -484,20 +484,18 @@ export const generateRootFolderIcons = async (
     return logger.error('Invalid color code for root folder icons');
   }
 
-  const rootFolderIcon =
-    'M16,5A11,11,0,1,1,5,16,11.01245,11.01245,0,0,1,16,5m0-3A14,14,0,1,0,30,16,14,14,0,0,0,16,2Zm0,8a6,6,0,1,0,6,6A6,6,0,0,0,16,10Z';
-  const rootFolderIconOpen =
-    'M16,5A11,11,0,1,1,5,16,11.01245,11.01245,0,0,1,16,5m0-3A14,14,0,1,0,30,16,14,14,0,0,0,16,2Z';
+  const rootFolderIconOuter = `<circle cx="8" cy="8" r="6" fill="none" stroke="${color}" stroke-width="2"/>`;
```

Comment:
Using circles is actually a nice idea and improves the quality of the default root folder icons, I like it :)